spec/factories.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :post do
+    title { "sampletitle" }
+    link { "https://example.com" }
+  end
+
+  factory :user do
+    email { "example@example.com" }
+  end
```

Comment:
This is small enough that it's okay all in one, but usually we'll do each factory in a different file. Next time you create a model, try moving these out into their own files. See https://github.com/vigetlabs/nest-app/tree/master/spec/factories as an example.